aten/src/ATen/cuda/Atomic.cuh

```diff
@@ -154,43 +154,11 @@ struct Atomic##NAME##IntegerImpl<T, 8> {
 };
 
 
-# define GPU_ATOMIC_INTEGER(NAME, OP)                                                                                  \
-static inline __device__ void gpuAtomic##NAME(uint8_t *address, uint8_t val) {                                         \
-Atomic##NAME##IntegerImpl<uint8_t, sizeof(uint8_t)>()(address,                                                         \
```

Comment:
very small nit: now that we have this we can technically use it for gpuAtomicAdd int8, uint8 and int16 if we want to, not sure what you think of this

Reply:
Good point! I missed that. We can do it in a followup.